app/src/main/res/values/strings.xml

```diff
@@ -377,6 +379,8 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
     <string name="share_via_another_app">Share via another App</string>
     <string name="msg_widechat_no_chats_to_display">You can get started by searching for people to chat with or by inviting your friends and family to get the app too.</string>
     <string name="msg_widechat_welcome_to_app">Welcome to Viasat Connect.</string>
+    <string name="new_group">New Group</string>
+    <string name="new_contact">New Contact</string>
```

Comment:
Any chance you can add spanish translations as well?  If that's not possible we can just make a story about it and do it later...

Reply:
I can google translate and then you can verify perhaps?

Reply:
Google translate is fine for now.... thanks!

Reply:
Done